### PR TITLE
fix: [sc-159] Error when getting events

### PR DESCRIPTION
**Pull request by gubbih**
Story details: https://app.shortcut.com/5-semester/story/159

### CODE REVIEW

File: src/firebase/events.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ export async function getAllEventsByOrgId(orgId: number): Promise<Event[]> {
     events.push(data);
   });
   if (!events.length) {
-    throw new BaseError('No events found', 404);
+    return events as Event[];
   }
```
Comment (mjrj97) [inline src/firebase/events.ts:54]: Does the same as line 55. Just remove

**Review by mjrj97 — CHANGES_REQUESTED** [review]

File: src/firebase/events.ts (hunk 1 above)
Comment (gubbih) [reply src/firebase/events.ts:54]: yea thought of that, the if is useless

Comment (mjrj97) [reply src/firebase/events.ts:54]: 👏

**Review by mjrj97 — APPROVED** [review]
So fresh so clean clean
https://www.youtube.com/watch?v=-JfEJq56IwI

**Pull request merged**